calibration/CTDMOG/CGINS-CTDMOG-13433__20161021.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13433,CC_ptempa0,1.483492e+002,
 37-13433,CC_ptempa1,-7.449100e-002,
 37-13433,CC_ptempa2,2.198810e-006,
-37-13433,CC_wbotc,3.1020e-007,
+37-13433,CC_wbotc,3.101952e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT